src/constraints/BooleanConstraints.ts

```diff
@@ -2,18 +2,20 @@ import { ConstraintError } from '../lib/errors/ConstraintError';
 import { Result } from '../lib/Result';
 import type { IConstraint } from './base/IConstraint';
 
+export type BooleanConstraintName = `s.boolean.${boolean}`;
+
 export const booleanTrue: IConstraint<boolean, true> = {
 	run(input: boolean) {
 		return input //
 			? Result.ok(input)
-			: Result.err(new ConstraintError('booleanTrue', 'Expected boolean to be true, but received false', input, true));
+			: Result.err(new ConstraintError('s.boolean.true', 'Invalid boolean value', input, 'expected'));
```

Comment:
How do these errors look for bool?

Reply:
![image](https://user-images.githubusercontent.com/24852502/152516503-e93f6e12-fb16-4b70-b116-59ccfbceb196.png)